src/app/pages/vehicle.cpp

```diff
@@ -54,6 +63,15 @@ Gauge::Gauge(units_t units, QFont value_font, QFont unit_font, Gauge::Orientatio
     layout->addWidget(unit_label);
 }
 
+void Gauge::can_callback(QByteArray payload){
+    Response resp = Response(payload);
+    for(auto cmd : cmds){
+        if(cmd.frame.payload().at(2) == resp.PID){
+            value_label->setText(this->format_value(this->decoder(cmd.decoder(resp), this->si)));
```

Comment:
this ends up seg faulting no? `value_label` (the class member) is no longer set, since it looks like its just a local var in the constructor

Reply:
yea I accidentally diffed to develop a bit too much